src/locales.py

```diff
@@ -120,6 +120,11 @@ class Locale:
     ChoosePrinterDirectory = "Choose printer name"
     AddNewPrinter = "Add new printer"
     DefaultPrinterWarn = "Be aware that you are using default printer. New data might be removed after update. We recommend to create new printer and calibrate it."
+    CheckUpdates = "Check for updates"
+    ProjectUpdate = "Project update"
+    SettingsUpdate = "The current project settings format is outdated. This may lead to errors in the program. Update format? When updating, some settings will be set to default!"
```

Comment:
"The current project's settings are outdated. This may lead to errors in the program. Do you want to update? New setting's fields will be set to default values, please recheck."